src/Adapter/MSTestAdapter.PlatformServices/Extensions/ExceptionExtensions.cs

```diff
@@ -13,22 +13,18 @@ internal static class ExceptionExtensions
     /// </summary>
     /// <param name="exception"> The exception. </param>
     /// <returns> Custom exception message that includes inner exceptions. </returns>
-    internal static string GetExceptionMessage(this Exception? exception)
+    internal static string GetExceptionMessage(this Exception exception)
     {
-        if (exception == null)
-        {
-            return string.Empty;
-        }
-
-        string exceptionString = exception.Message;
+        var builder = new StringBuilder(exception.Message);
         Exception? inner = exception.InnerException;
         while (inner != null)
         {
-            exceptionString += Environment.NewLine + inner.Message;
+            builder.AppendLine();
+            builder.AppendLine(inner.Message);
```

Comment:
This appends an extra new line compared to the old logic.

Reply:
oops. will fix